### DISCUSSION TITLE
Yellow Wifi Icon

### DISCUSSION THREAD

**Discussion by bkbartk** [Blackymas/NSPanel_HA_Blueprint#2217] [Q&A] [answered]
What does a yellow WiFi icon mean?
since I made some migrations to my network I sometimes have a yellow WiFi icon instead of White.
I cannot find what it means and if I should do something about it.

**Comment by edwardtfn** [comment, marked as answer]
Yellow icon means your Wi-Fi signal is low.

**Reply by bkbartk** [reply]
thnx, I expected something like that,
that would mean devices do not always connect to the closest access point, this is something I have to figure out, but is unrelated to this component.
